docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/02-Environment Setup.mdx

```diff
@@ -33,7 +33,7 @@ This document covers the essential setup required to start building on Orca’s
     Install the necessary packages:
 
     ```bash
-    npm install typescript @orca-so/whirlpools @solana/web3.js@rc
+    npm install typescript @orca-so/whirlpools @solana/web3.js@2
```

Comment:
Is version 2 now recommended (meaning you can remove the tag)?

Reply:
`npm install @solana/web3.js` installs v1.98.0